realm/src/main/java/io/realm/RealmResults.java

```diff
@@ -317,9 +317,11 @@ public int size() {
     /**
      * Find the minimum value of a field.
      *
-     * @param fieldName   The field to look for a minimum on. Only int, float, and double
+     * @param fieldName   The field to look for a minimum on. Only integer and floating-point fields
      *                    are supported.
-     * @return            The minimum value.
+     * @return            If no objects exist or they all have {@code null} as the value for the given
+     *                    field, {@code null} will be returned. Otherwise the minimum value is returned.
+     *                    In calculations of aggregate functions, objects with {@code null} values will be ignored.
```

Comment:
This sentance is not relevant for this method. Sounds more like a top-level description on the RealmQuery object.